connect/2.1/Dockerfile

```diff
@@ -14,6 +14,7 @@ ENV DEBEZIUM_VERSION="2.0.0.Final" \
     ORACLE_MD5=2ff310589236a823614a95ec0ce15467 \
     DB2_MD5=42ec53aa26ff6626a7a2a67cdc9a2223 \
     VITESS_MD5=f45966b21a4d548108cd4006421406cb \
+    KCRESTEXT_MD5=986c2196db9ca076117198f00e8962ac \
```

Comment:
Looking for suggestions on how to move forward here. I do not see a DBZ v2.1 artifact for the KC REST Extension at https://repo1.maven.org/maven2/io/debezium/debezium-connect-rest-extension/ and this is from the v2.0.0 artifact (https://repo1.maven.org/maven2/io/debezium/debezium-connect-rest-extension/2.0.0.Final/debezium-connect-rest-extension-2.0.0.Final.jar.md5). This will cause the build to fail as the scripts are expecting an artifact that matches the DBZ version ($DEBEZIUM_VERSION) of the image being built.

Reply:
Nevermind, the variable is still pointing to v2.0 as a 2.1 release hasn't been cut yet I suppose.